### DISCUSSION TITLE
薯条当夜宵真的健康吗？

### DISCUSSION THREAD

**Discussion by cpu644** [share121/inter-knot#5449] [其他]
![IMG_6928](https://github.com/user-attachments/assets/1a527f68-fed2-4891-a4ea-1b43b5514a6a)
![IMG_6922](https://github.com/user-attachments/assets/137d5371-ef7a-43ad-bd30-c420f8867761)

入夜，曾日の喧嚣不再。吃点东西吧，脑子如是说。离开温暖的小棉被- 扯出个烤盘，扔片披萨上去，便成了个自封的御膳厨子。
但品味美食时，可何曾想过后果？答曰：没想过。
发动技能：思考！
心情-1（体重+1）
这下便无奈了。
敲开音箱，扛起小吉他，甩半个小时的头，这才作罢，一头栽到床上。不知道有没有用，但至少爽了。
本来昨天晚上一点发的，但是没发成功故再试试。

**Comment by KELERSAMA** [comment]
没事 吃完之后就心情恢复至100% 体重加5了 
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Reply by cpu644** [reply]
越吃越伤心。体重管理失败 心情值降到40% 体重+5了。

**Comment by Orangered81** [comment]
薯条有助于减少压力值，清除侵蚀症状<img src="https://img-static.mihoyo.com/communityweb/upload/068a8b611fae7cde48470ebfe5e21847.png" alt="米游姬得意" width="75" >

**Comment by StellaBigHero** [comment]
压力-30，体重+5，受到侵蚀【胃部重担】
